### ISSUE TITLE
Is CUDA required for both training and testing? & Other Questions

### ISSUE THREAD

**Issue by MelodyJoy7** [ViTAE-Transformer/ViTPose#88]
Hi! I appreciate your wonderful work!

I am new to this field and would love to hear all of your suggestions.

## 1. Is CUDA required for testing?
I've been trying to run the pretrained models and demos, and could never succeed until I used a platform which provides CUDA and GPU, and at least the demo worked.

I was wondering if CUDA is required for me to _test_ the model on my own datasets (not training the models)? I might not have access to CUDA later on so this would be a great concern for me.

## 2. What's the difference of _Pretrained PATH_ and _Checkpoint PATH_?
For instance, if I want to run the ViTPose-S model for Wholebody dataset, I assume both Pretrained PATH and Checkpoint PATH refer to [the small size MAE pre-trained model](https://onedrive.live.com/?authkey=%21AHohY4eAye4I2Mo&id=E534267B85818129%2125497&cid=E534267B85818129&parId=root&parQt=sharedby&o=OneUp) as indicated on the [homepage](https://github.com/ViTAE-Transformer/ViTPose). Or are they two different things?

Below is the sample code provided on the homepage:

**training**

`bash tools/dist_train.sh <Config PATH> <NUM GPUs> --cfg-options model.pretrained=<Pretrained PATH> --seed 0`

**testing**

`bash tools/dist_test.sh <Config PATH> <Checkpoint PATH> <NUM GPUs>`

## 3. NUM GPUs
Where could I find the information of the number of GPUs I should fill in? As in the above mentioned code `<NUM GPUs>`.

Thank you all! I am looking forward to your reply. May the smiley face bring you a happy day 😃
